### ISSUE TITLE
Staging Review finding: Consider calculating number of remaining items

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82520]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** List and Loop
**Feature:** Reusable multi-page

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In this sample list-and-loop, there's a maximum of 5 items that can be added. On the review page when asked "Do you have another employer to report?" the helper text says "You can add up to 5."

Strictly speaking, this provides users with all of the information they need to complete the task. But to reduce cognitive load, I would suggest helper text telling the user how many remaining items they can add, rather than repeating the maximum number.

## Recommended action

Helper text that reads "You can add up to 4 more" or something similar could reduce cognitive load, especially if the max number is large and users might lose track of how many they've added. (eg. max is 20, need to count all 17 I've added to know that I have 3 left.) 

As noted, this is a suggestion for improvement and not an accessibility barrier. I'm not sure how much of this is intended to be left to VFS teams to implement and how much is supposed to be copied out-of-the-box, but I think it would be helpful to establish it as a pattern. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by rhasselle-oddball** [comment]
can be tested at https://staging.va.gov/mock-form-patterns/introduction

**Comment by jeana-adhoc** [comment]
This was addressed! 

<img width="1170" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVZIQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f5c47038684b9d54863a56d2f171b80aec833b77/image.png" alt="image.png" />
